src/managed/Microsoft.NET.HostModel/AppHost/HostWriter.cs

```diff
@@ -43,6 +45,16 @@ public static class HostWriter
             }
 
             BinaryUtils.CopyFile(appHostSourceFilePath, appHostDestinationFilePath);
+
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                var filePermissionOctal = Convert.ToInt32("755", 8); // -rwxr-xr-x
```

Comment:
I suggest writing this as an integer constant

Reply:
IMO, this is not a hot path, more readable and consistent with tests.

Reply:
OK, that's fine.